pkg/metrics/nextgengrafana/tidb_with_keyspace_name.json

```diff
@@ -126,38 +132,47 @@
             "alertThreshold": true
           },
           "percentage": false,
-          "pluginVersion": "7.5.11",
+          "pluginVersion": "7.5.17",
           "pointradius": 5,
           "points": false,
           "renderer": "flot",
+          "scopedVars": {
+            "keyspace_name": {
+              "selected": true,
+              "text": "admin",
+              "value": "admin"
```

Comment:
What's the value "admin"?

Reply:
By the way I noticed that the `scopedVars` field shows up only in several panels but not all. Is this expected?